code/modules/reagents/reagent_containers/spray.dm

```diff
@@ -23,6 +23,7 @@
 	amount_per_transfer_from_this = 5
 	volume = 250
 	possible_transfer_amounts = list(5,10,15,20,25,30,50,100)
+	APTFT_altclick = FALSE
```

Comment:
This only stops it from being toggled through altclick, they will still be able to use the verb.
I guess it's time to refactor this thing a little more..